src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -499,33 +499,35 @@ namespace Windows.ApplicationModel.DataTransfer
 #endif
     static class DataTransferManagerInterop
     {
-        private static readonly global::System.Guid IDataTransferManagerInterop_IID = new global::System.Guid("3A3DCD6C-3EAB-43DC-BCDE-45671CE800C8");
+        private static readonly global::System.Guid IDataTransferManagerInterop_IID = new global::System.Guid(0x3A3DCD6C, 0x3EAB, 0x43DC, 0xBC, 0xDE, 0x45, 0x67, 0x1C, 0xE8, 0x00, 0xC8);

```

Comment:
This is wrong endianess.

Reply:
Correct me if I am wrong, but this uses the format where the values goes as parameters and not in the byte array where we need to flip it for endianness, so I think this is correct right?

https://learn.microsoft.com/en-us/dotnet/api/system.guid.-ctor?view=net-8.0#system-guid-ctor(system-int32-system-int16-system-int16-system-byte-system-byte-system-byte-system-byte-system-byte-system-byte-system-byte-system-byte)